pkg/controller/sparkapplication/submission_test.go

```diff
@@ -398,22 +398,40 @@ func TestPopulateLabels_Driver_Executor(t *testing.T) {
 		t.Fatal(err)
 	}
 	assert.Equal(t, 5, len(driverOptions))
-	assert.Equal(t, fmt.Sprintf(SparkDriverLabelAnnotationTemplate, "app-name", "spark-test"), driverOptions[0])
-	assert.Equal(t, fmt.Sprintf(SparkDriverLabelAnnotationTemplate, "launched-by-spark-operator", strconv.FormatBool(true)), driverOptions[1])
-	assert.Equal(t, fmt.Sprintf(SparkDriverLabelAnnotationTemplate, "submission-id", submissionID), driverOptions[2])
-	assert.Equal(t, fmt.Sprintf(SparkDriverLabelTemplate, AppLabelKey, AppLabelValue), driverOptions[3])
-	assert.Equal(t, fmt.Sprintf(SparkDriverLabelTemplate, DriverLabelKey, DriverLabelValue), driverOptions[4])
+	ExpectedDriverLabels := []string{
```

Comment:
Variables should start with a lower case letter.

Reply:
Done.